### PR TITLE
adds password protection to full site

### LINKED ISSUE #139: Password protect site for embargoed release?
https://www.gatsbyjs.com/plugins/@mkitio/gatsby-theme-password-protect/ looks promising

### LINKED ISSUE #216: Bring back the password idea (for reports page only)

### PR DESCRIPTION
**Pull request by Frijol**
This pull request causes the site to request a password for access.

Some VERY IMPORTANT notes:
* The reports are already hosted at public URLs on EDGI's public Github
* This is an open source website that has the URLs publicly hosted
* Because of this stuff, I'm assuming anybody who reeeally wants to access our reports & has a basic knowledge of Github will just do so. This is "pseudoprotected", not actually secure
* Because of ^ I'm including the password in plaintext here because it's a lot easier than a more secure implementation and we're already deeply insecure.

Basically, this is enough that people won't casually stumble across our site. Anybody who goes to the URL will be asked for a password, and that will probably keep it from getting shared widely. But it's not really more than that.

### CODE REVIEW

**Comment by Frijol** [comment]
Decided not to do this

**Comment by Frijol** [comment]
reviving this PR in honor of #216

**Comment by Frijol** [comment]
This should work but doesn't show up in localhost, but I remember it had trouble propagating last time, so will merge. Possible that the /EEW-Website/ thing will disrupt as well in deployment (which I can work around by adding an additional path but overall hope to fix)

**Comment by Frijol** [comment]
as far as I can tell this had no effect

**Comment by Frijol** [comment]
docs, for reference https://www.gatsbyjs.com/plugins/@mkitio/gatsby-theme-password-protect/
